### DISCUSSION TITLE
How to use Pie Graph control

### DISCUSSION THREAD

**Discussion by ZXueW** [frangoteam/FUXA#1498] [Q&A] [answered]
Hi, would anyone please know whether the feature of pie graph control is available? Seems that the settings cannot be changed from the picture below. Gratefull to show me the usage of this control, where I can change the legend and where to bind the device tags. The version is 1.2.1. Thanks.
<img width="404" alt="piegraph" src="https://github.com/user-attachments/assets/4942d314-2e7b-434d-b9e3-7449fad91eed">

**Comment by unocelli** [comment, marked as answer]
Hi, you can use like a bar chart - only value
![image](https://github.com/user-attachments/assets/2d55ffec-98c8-4e22-9722-e1e858eab904)

![image](https://github.com/user-attachments/assets/351258f2-728c-4b07-a86a-dafb2bcf9a70)

![pie-2](https://github.com/user-attachments/assets/ab500c8a-8d68-49ae-a50c-ab93c6dd2979)

**Reply by ZXueW** [reply]
Got it, thank you very much.
